scanpipe/pipelines/deploy_to_develop.py

```diff
@@ -134,6 +135,14 @@ def fingerprint_codebase_directories(self):
         """Compute directory fingerprints for matching"""
         matchcode.fingerprint_codebase_directories(self.project, to_codebase_only=True)
 
+    def flag_whitespace_files(self):
+        """
+        Flag whitespace files with size less than or equal
+        to 1 byte as ignored.
+        """
+        qs = self.project.codebaseresources.files().filter(size__lte=1)
```

Comment:
How can you guarantee the content is whitespace when the size is 1 or less?

Also, what's your take on https://github.com/nexB/scancode.io/issues/969#issuecomment-1760828076 ?
What would be a minimum size to not be flagged as ignored?

Reply:
> How can you guarantee the content is whitespace when the size is 1 or less?

In our use case, we found that files with whitespace characters were 1 bytes files, and it may not be true all the time.

> Also, what's your take on [#969 (comment)](https://github.com/nexB/scancode.io/issues/969#issuecomment-1760828076) ?

It would be impractical to look inside each and every file content. IMO we can filter out the files which are 1 byte or less in size and then use regex to confirm that indeed the content inside the files is just whitespace characters.


> What would be a minimum size to not be flagged as ignored?

I don't have numbers on this ATM, but I guess we are good with not ignoring anything above 1 byte, at least for now.

Reply:
> It would be impractical to look inside each and every file content.

I think you are missing the point. I'm just wondering how valuable is to scan 2 bytes files, 3 bytes files, etc...
Is there any value in analyzing files that small, are there any licenses/copyrights to be detected in 2-byte files?

>  I guess we are good with not ignoring anything above 1 byte

Alright then.